### ISSUE TITLE
natverse install broken: remotes internal functions have been renamed

### ISSUE THREAD

**Issue by jefferis** [natverse/natverse#19]
Hi @SridharJagannathan,

I hadn't noticed that you are using internal remotes functions here:

https://github.com/natverse/natverse/blob/3dbb6d03353e227144064966b41f0e3818a0f6fc/R/utils_warnings.R#L8-L17

This was a bit dangerous and unfortunately it looks like the [latest version of remotes](https://cran.r-project.org/web/packages/remotes/news/news.html) has changed some of these functions. 

@dbock just tried to install and got this:

```
─  building 'natverse_0.2.1.tar.gz'
* installing source package 'natverse' ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error in get(fun, envir = asNamespace(pkg), inherits = FALSE) : 
  object 'parse_one_remote' not found
Error: unable to load R code in package 'natverse'
Execution halted
ERROR: lazy loading failed for package 'natverse'
* removing '/Library/Frameworks/R.framework/Versions/4.0/Resources/library/natverse'
ℹ Load the full natverse with library(natverse)
Warning message:
In i.p(...) :
  installation of package '/var/folders/7w/y7qvbf5x7wldcf52hgft81pw0000gq/T//Rtmp0wGyvJ/file6a16d7e8ec7/natverse_0.2.1.tar.gz' had non-zero exit status
```
It looks to me that this will break natverse installation for all new users now, so it's a priority to fix.

**Comment by jefferis** [comment]
@dbock I think you can do

```
remotes::install_github("r-lib/remotes@v2.1.1")
```

restart R

```
natmanager::install("natverse", upgrade.dependencies="never")
```

I think the never may be required to stop it trying to upgrade remotes again ...

**Comment by SridharJagannathan** [comment]
Just adding some notes here:

```
> natmanager::install("natverse", dependencies = FALSE)
```
Produces
```
....

Error in get(fun, envir = asNamespace(pkg), inherits = FALSE) : 
  object 'parse_one_remote' not found
Error: unable to load R code in package ‘natverse’
Execution halted
ERROR: lazy loading failed for package ‘natverse’
* removing ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/natverse’
* restoring previous ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/natverse’
ℹ Load the full natverse with `library(natverse)`
Warning message:
In i.p(...) :
  installation of package ‘/var/folders/6x/d_rd5yr937vbq8f46d4yzrsw0000gn/T//RtmpG4NLY4/fileab422bda90f3/natverse_0.2.1.tar.gz’ had non-zero exit status
```

**Comment by jefferis** [comment]
Closed by #20
